src/controllers/controllers/controller.py

```diff
@@ -136,7 +137,17 @@ def omega_to_ackerman(self, omega, linear_v, L):
 
         return delta
 
-
+    def vel_mod(self, linear):
```

Comment:
I think it's better to restrict the models, max and min actions rather than clip it. Like do the match backwards to find the maximum and minimum angular velocity - if it can be done. Don't really know. Cause then there's a whole range of values that the RL will try to do that won't be reflected in the state

Reply:
Will fix this and comment below tomorrow, then can merge

Reply:
Can we merge as is and I'll make a new branch to make the bigger changes for this